### ISSUE TITLE
Helper functions for preparing sunpy.map.Map objects for pfsspy.Input

### ISSUE THREAD

**Issue by STBadman** [dstansby/pfsspy#169]
**Roll Function**
Take sunpy.map.Map instance, verify is synoptic map, edit Map.data (inplace?) : roll carrington longitude = 0 to LH edge of map

**Interpolate to sinlat function**
Take sunpy.map.Map instance, verify is synoptic map, verify latitudinal binning is not sinlat, edit Map.data (inplace?) : interpolate data to uniform binning in sinlat (same dimensions as previous?)

**Comment by STBadman** [comment]
@dstansby would another helper function titled something like load_adapt be appropriate in this development? It would take a path to an adapt .fts file as input and returns a MapSequence as in #170 , I'm writing it for local convenience but can submit it in a PR if you think it's useful?

**Comment by dstansby** [comment]
Yes, that sounds useful!

**Comment by dstansby** [comment]
@STBadman I've got the CEA reprojection working well here: https://github.com/dstansby/pfsspy/pull/199, if you want to add any comments on the PR feel free (I still have to expand the gallery example). I'm thinking about how to test this. Total flux should be conserved, which I will add as a test, but do you have any ideas for other tests that could be added?

**Comment by STBadman** [comment]
Sorry I didn't get to giving feedback yet but I saw the PR was closed, are you adding it to a future PR?

**Comment by dstansby** [comment]
Woops, I didn't mean to close that... will re-open it.

**Comment by dstansby** [comment]
I think the status here is there still needs to be a roll function?

**Comment by STBadman** [comment]
I think that's right. I think in our last correspondence on this you mentioned this could be accomplished within `reproject`. Are you familiar enough with `reproject` to do this quickly? I'm happy to take a look otherwise.

**Comment by dstansby** [comment]
The `car_to_cea` function in `pfsspy/utils.py` uses a reproject approach. The nice thing is, if you know how to modify the FITS header to get the desired result, the reproject option is pretty easy to modify. So if you want to have a go, it should be pretty easy to copy `car_to_cea`, and just change how the FITS header is modified. Make sure to grab the latest version of the code, I just cleaned up `car_to_cea`, and I've also renamed the `master` branch to `main`.
